addons/website_profile/controllers/main.py

```diff
@@ -73,40 +73,6 @@ def _prepare_user_profile_values(self, user, **post):
             'edit_button_url_param': '',
         }
 
-    @http.route([
-        '/profile/avatar/<int:user_id>',
-    ], type='http', auth="public", website=True, sitemap=False)
-    def get_user_profile_avatar(self, user_id, field='image_256', width=0, height=0, crop=False, **post):
-        if field not in ('image_128', 'image_256'):
-            return werkzeug.exceptions.Forbidden()
-
-        can_sudo = self._check_avatar_access(user_id, **post)
```

Comment:
Don't we still need to actually keep this route for the purpose of this access check?

Note that there is a similar one in mail with access check to thread, and we would actually need one (currently missing) for livechat to do also a similar thing.